redwood-treehouse-host/src/iosMain/kotlin/app/cash/redwood/treehouse/TreehouseUIKitView.kt

```diff
@@ -17,30 +17,17 @@ package app.cash.redwood.treehouse
 
 import app.cash.redwood.treehouse.TreehouseView.ReadyForContentChangeListener
 import app.cash.redwood.treehouse.TreehouseView.WidgetSystem
-import app.cash.redwood.ui.Default
-import app.cash.redwood.ui.Density
-import app.cash.redwood.ui.Margin
-import app.cash.redwood.ui.Size
-import app.cash.redwood.ui.UiConfiguration
 import app.cash.redwood.widget.UIViewChildren
-import app.cash.redwood.widget.Widget
-import kotlinx.cinterop.CValue
 import kotlinx.cinterop.cValue
-import kotlinx.cinterop.useContents
-import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.StateFlow
-import platform.CoreGraphics.CGRect
 import platform.CoreGraphics.CGRectZero
-import platform.UIKit.UIApplication
 import platform.UIKit.UITraitCollection
-import platform.UIKit.UIUserInterfaceStyle.UIUserInterfaceStyleDark
 import platform.UIKit.UIView
 
 @ObjCName("TreehouseUIKitView", exact = true)
 public class TreehouseUIKitView(
   override val widgetSystem: WidgetSystem,
-) : TreehouseView {
-  public val view: UIView = RootUiView(this)
+) : TreehouseView, RedwoodUIKitView() {
```

Comment:
Future change: We probably can drop the "Kit" from these names. Feels out of place and doesn't fit the normal naming pattern of their types.

Reply:
Should we also rename `TreehouseWidgetView`? All the implementations of `TreehouseView` are "widget" views. Something like `TreehouseViewView` would be awful but consistent. `TreehouseAndroidView` perhaps?

Reply:
Actually, not a big fan of `TreehouseAndroidView` either. I'm drawing a blank on a better name tbh

Reply:
`TreehouseLayout`?

Reply:
Is "Layout" Android specific enough?

Reply:
Renamed to `TreehouseLayout` in https://github.com/cashapp/redwood/pull/1451.
Renamed to `TreehouseUIView` in https://github.com/cashapp/redwood/pull/1452.